contributors/devel/contributing-api-changes.md

```diff
@@ -0,0 +1,134 @@
+# Guidelines for contributing api changes
+
+## Overview
+
+Modifications to the core Kubernetes api have a material impact on the Kubernetes project, and
+undergo thorough review and discussion before they are accepted.  This document outlines the
+review process and what is expected of api contributors.
+
+## Changes that qualify as api changes
+
+Any changes to existing or new apis are subject to the process outlined in this document.
+
+The following qualify as api changes:
+
+- Adding, removing, or modifying api types (e.g. through types.go, or by defining a new annotation)
+- Changing the version of an existing type (e.g. v1alpha1 to v1beta1)
+- Adding, removing, or modifying subresources for existing types. (e.g. /scale)
+- Material changes to how fields on existing types are interpreted / used
+
+The following on their own do not qualify as api changes, but should follow the process defined by the related
+[special interest groups].
+
+- Adding or changing a command or flag to kubectl: sig-cli
```

Comment:
This is where it would be useful to link back to a more general proposal process.